test/package-managers-tests/basic-pnpm/test/index.test.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-env jest */
+import { execSync } from 'child_process'
+import fs from 'fs-extra'
+import path from 'path'
+
+jest.setTimeout(100 * 1000)
+
+beforeAll(async () => {
+  await clean()
+})
+
+afterAll(async () => {
+  await clean()
+})
+
+test('pnpm installs', async () => {
+  const pnpm = getStdout('yarn bin pnpm')
+  exec(pnpm + ' init -y')
+  exec(pnpm + ' add next react react-dom')
+  await useLocalNextjs()
+  // exec('pnpm install')
+})
+
+test('nextjs builds with pnpm', () => {
+  const pnpx = getStdout(`yarn bin pnpx`)
+  exec(pnpx + ' next build')
+})
+
+const exec = (cmd) => {
+  return execSync(cmd, {
+    env: process.env,
+    shell: true,
+    stdio: 'inherit',
+    cwd: path.resolve(__dirname, '..'),
+  })
+}
+
+const getStdout = (cmd) => {
+  return execSync(cmd, {
+    env: process.env,
+    shell: true,
+    stdio: 'pipe',
+  })
+    .toString()
+    .trim()
+}
+
+async function useLocalNextjs() {
```

Comment:
This function looks incorrect as it only considers packages/next but Next.js has a dependency of other packages in the monorepo as well which would not be copied, so this would break eventually.

Reply:
Can you fix that in a follow-up PR as I'll merge this one.